data/sources/00_areatree

```diff
@@ -542,13 +562,14 @@
       5613:Finger 13 (BT13):
     -5622:Sportanlage Garching / Dusch Container:
     5701:Munich School of BioEngineering, IMETUM:
+    5801:Leibnitz-Rechenzenturm (LRZ):
```

Comment:
should we add the id `lrz`?

Reply:
I am not sure if 5801 is the whole LRZ (my assumption is it's not). The ID `lrz` would make sense if we know there is a parent with multiple buildings belonging to it. But with just one child building I think it doesn't make sense at this moment